books/dev-books/HTML and CSS/htmlcss.md

```diff
@@ -11,6 +11,7 @@ Please kindly follow this structure when you are contributing
 
 ### C
 - [CSS3 The Missing Manual](https://drive.google.com/file/d/1L3tVGwLb_ZMbdfq7gmV75Ixl96CuOty0/view?usp=share_link)
+- [CSS Notes for Professionals](https://goalkicker.com/CSSBook/)
 
 - [CSSS SECRETS: BETTER SOLUTIONS TO EVERYDAY CSS PROBLEMS (INTERMEDIATE LEVEL)](https://www.pdfdrive.com/css-secrets-better-solutions-to-everyday-web-design-problems-e177990723.html)
 
```

Comment:
If you could help us add a *###H* section in your next PR, it would be super awesome 😎